### ISSUE TITLE
Colour contrast on minor alert banner

### ISSUE THREAD

**Issue by andybroomfield** [localgovdrupal/localgov_alert_banner#238]
See https://github.com/localgovdrupal/localgov_base/issues/375

Maybe make this one solid black, as the other alerts are white border and go solid white on focus / hover.

**Comment by andybroomfield** [comment]
Whilst here, I notice the alert banners dont use hand cursor on hover, so a `pointer: cursor;` needs to be added.

**Comment by thiyagarajan17** [comment]
Assign issue. would be glad to work on it

**Comment by andybroomfield** [comment]
Yes please, thanks @thiyagarajan17
